### ISSUE TITLE
Extracto de suelo saturado

### ISSUE THREAD

**Issue by dariorodriguez** [INTA-Suelos/SiSinta#86]
Estimados: estoy pasando unos perfiles del sur de Buenos Aires y al ser salinos vamos a tener que agregar una tabla extra en "Analíticos" para estos suelos.

Esto surge ya que en los suelos salinos se hacen análisis complementarios sobre un extracto. Quisiera saber como subo la tabla (en que formato, vá)

**Comment by mauriciopasquier** [comment]
On Tue, Mar 10, 2015 at 08:55:16AM -0700, Dario Martin Rodriguez wrote:
> Estimados: estoy pasando unos perfiles del sur de Buenos Aires y al ser
> salinos vamos a tener que agregar una tabla extra en "Analíticos" para estos
> suelos.
> 
> Esto surge ya que en los suelos salinos se hacen análisis complementarios
> sobre un extracto. Quisiera saber como subo la tabla (en que formato, vá)

una tabla completamente nueva o una fila en la tabla que ya hay?

--

**Comment by dariorodriguez** [comment]
Sería una tabla anexa a continuación de la de datos analíticos, pero sólo se habilitaría la carga para suelos salinos.

Se me ocurre por ej. que al final de la carga de analíticos pregunte si se quiere cargar el "Extracto del suelo saturado" y en esa caso se habilite

**Comment by angelini75** [comment]
@mauriciopasquier Se trata de una tabla con la misma estructura que analíticos. Al igual que Analíticos, tiene que tomar los horizontes ya cargados. No recuerdo cuantos datos son, pero quizás @dariorodriguez puede listar nombre y tipo de propiedades, por ejemplo:

Variable, tipo, unidades, rango de valores
Conductividad, numérico, mhos/em, 0,01-200,00
Salinidad, cualitativo, NA , alta; media; baja

**Comment by dariorodriguez** [comment]
Sí...yo la tabla se la doy, pero quería saber como cargarla para que le sea
más fácil

El 11 de marzo de 2015, 5:13, Marcos Angelini <notifications@github.com>
escribió:

> @mauriciopasquier <https://github.com/mauriciopasquier> Se trata de una
> tabla con la misma estructura que analíticos. Al igual que Analíticos,
> tiene que tomar los horizontes ya cargados. No recuerdo cuantos datos son,
> pero quizás @dariorodriguez <https://github.com/dariorodriguez> puede
> listar nombre y tipo de propiedades, por ejemplo:
>
> Variable, tipo, unidades, rango de valores
> Conductividad, numérico, mhos/em, 0,01-200,00
> Salinidad, cualitativo, NA , alta; media; baja
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/INTA-Suelos/SiSinta/issues/86#issuecomment-78220744>.
>



-- 
Darío

**Comment by midraed** [comment]
No, Darío no se puede cargar la tabla en excel o similar. Mauricio tiene
que generar los campos en la base de datos, y luego las relaciones, y luego
la interfaz. No es muy complciado pero le va a llevar tiempo.

Para ir acercandonos al problema yo creo que podrias ir mostrandonos
cuantos campos son y como se relaciona con la otra tabla. Pero, creo que
mauricio deberia enfocarse en los otros pendientes. Estos es una
funcionalidad nueva y que no estaba planeada.

o/

**Comment by dariorodriguez** [comment]
Bueno, eso como les parezca, ya que es cierto que no estaba planeado y ya
lo estamos jodiendo bastante en estos días.
Pero para que lo vayan viendo sería una tabla anexa a la de analíticos,
donde a cada Horizonte (los mismos del perfil) se le miden unas 10
variables más

El 11 de marzo de 2015, 9:12, Guillermo Federico Olmedo <
notifications@github.com> escribió:

> No, Darío no se puede cargar la tabla en excel o similar. Mauricio tiene
> que generar los campos en la base de datos, y luego las relaciones, y luego
> la interfaz. No es muy complciado pero le va a llevar tiempo.
>
> Para ir acercandonos al problema yo creo que podrias ir mostrandonos
> cuantos campos son y como se relaciona con la otra tabla. Pero, creo que
> mauricio deberia enfocarse en los otros pendientes. Estos es una
> funcionalidad nueva y que no estaba planeada.
>
> o/
>
> El 11 de marzo de 2015, 9:08, Dario Martin Rodriguez <
> notifications@github.com> escribió:
>
> > Sí...yo la tabla se la doy, pero quería saber como cargarla para que le
> sea
> > más fácil
> >
> > El 11 de marzo de 2015, 5:13, Marcos Angelini <notifications@github.com>
> > escribió:
> >
> > > @mauriciopasquier <https://github.com/mauriciopasquier> Se trata de
> una
> > > tabla con la misma estructura que analíticos. Al igual que Analíticos,
> > > tiene que tomar los horizontes ya cargados. No recuerdo cuantos datos
> > son,
> > > pero quizás @dariorodriguez <https://github.com/dariorodriguez> puede
> > > listar nombre y tipo de propiedades, por ejemplo:
> > >
> > > Variable, tipo, unidades, rango de valores
> > > Conductividad, numérico, mhos/em, 0,01-200,00
> > > Salinidad, cualitativo, NA , alta; media; baja
> > >
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <
> https://github.com/INTA-Suelos/SiSinta/issues/86#issuecomment-78220744
> > >.
> > >
> >
> >
> >
> > --
> > Darío
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/INTA-Suelos/SiSinta/issues/86#issuecomment-78251148
> >.
>
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/INTA-Suelos/SiSinta/issues/86#issuecomment-78251565>.
>



-- 
Darío
